meta-lmp-base/recipes-security/optee/optee-os-fio.inc

```diff
@@ -12,6 +12,7 @@ DEPENDS = "python3-pyelftools-native dtc-native"
 DEPENDS:append:toolchain-clang = " compiler-rt"
 
 require optee-fio.inc
+include optee-os-lmp.inc
```

Comment:
One thing to check, this will end up generating configs for both optee-os-fio and optee-os-tadevkit right?

Not sure we can avoid that, unless we check for tadevkit in PN.

Reply:
@ricardosalveti Sure, tadevkit. I'm checking this.

Reply:
I think this is not a problem because the config is deployed as `${DEPLOYDIR}/${PN}-config-${MACHINE}-${PV}-${PR}` so `PN` is already there.

Reply:
Yup, won't conflict, but there is no need to have the ones from tadevkit (but ok if avoiding it makes the logic more complicated than what already is).

Reply:
> One thing to check, this will end up generating configs for both optee-os-fio and optee-os-tadevkit right?

This doesn't affect `optee-os-tadevkit` because that recipe redefines `deploy`:
```
do_deploy() {
    echo "Do not inherit do_deploy from optee-os."
}
```